azclishell/app.py

```diff
@@ -387,9 +387,9 @@ def example_repl(self, text, example, start_index):
     def _special_cases(self, text, cmd, outside):
         break_flag = False
         continue_flag = False
-        if 'az' in text:
+        if text and text.split()[0] == 'az':
             telemetry.track_ssg('az', text)
-            cmd = cmd.replace('az', '')
+            cmd = ' '.join(text.split()[1:])
```

Comment:
I think that's a good idea to have 'az' as a way to define an absolute path. However, I  want to strip the 'az' because I want to have people who are used to the CLI to always writing 'az vm create' to be able to do the same in the shell even though it would be better to just do 'vm create'